test/autoCompletion.test.ts

```diff
@@ -72,8 +74,8 @@ describe('Auto Completion Tests', () => {
             },
           },
         });
-        const content = '|:|';
-        const completion = parseSetup(content);
+        const content = '';
+        const completion = parseSetup(content, 0);
```

Comment:
Reverted this earlier change - a caret cannot be placed in a zero-length string, so use the old algorithm